docs/topics/data-analysis/data-analysis-visualization.md

```diff
@@ -0,0 +1,168 @@
+the [Kandy](https://kotlin.github.io/kandy/welcome.html) and [Kotlin DataFrame](https://kotlin.github.io/dataframe/gettingstarted.html) libraries.
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+   
+    > Alternatively, access the Kotlin Notebook plugin from **Settings/Preferences** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+    >
+    {type="tip"}
+
+3. Create a new notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. In your notebook, import the Kandy and Kotlin DataFrame libraries by running the following command:
+
+    ```kotlin
+    %use kandy
+    %use dataframe
+    ```
+
+## Create the DataFrame
+
+Start by creating the DataFrame containing the records to visualize. This DataFrame stores simulated numbers of the 
+monthly average temperature in three cities: Berlin, Madrid, and Caracas.
+
+Use the `dataFrameOf()` function from the Kotlin DataFrame library
+to generate the DataFrame. Run the following code snippet in Kotlin Notebook:
+
+```kotlin
+// The months variable stores a list with the 12 months of the year
+val months = listOf(
+    "January", "February",
+    "March", "April", "May",
+    "June", "July", "August",
+    "September", "October", "November",
+    "December"
+)
+// The tempBerlin, tempMadrid, and tempCaracas variables store a list with temperature values for each month
+val tempBerlin =
+    listOf(-0.5, 0.0, 4.8, 9.0, 14.3, 17.5, 19.2, 18.9, 14.5, 9.7, 4.7, 1.0)
+val tempMadrid =
+    listOf(6.3, 7.9, 11.2, 12.9, 16.7, 21.1, 24.7, 24.2, 20.3, 15.4, 9.9, 6.6)
+val tempCaracas =
+    listOf(27.5, 28.9, 29.6, 30.9, 31.7, 35.1, 33.8, 32.2, 31.3, 29.4, 28.9, 27.6)
+
+// The df variable stores a DataFrame of three columns, including records of months, temperature, and cities
+val df = dataFrameOf(
+    "Month" to months + months + months,
+    "Temperature" to tempBerlin + tempMadrid + tempCaracas,
+    "City" to List(12) { "Berlin" } + List(12) { "Madrid" } + List(12) { "Caracas" }
+)
+```
+
+Explore the structure of the new DataFrame by looking into the first four rows:
+
+```kotlin
+df.head(4)
+```
+
+You can see that the DataFrame has three columns: Month, Temperature, and City. 
+The first four rows of the DataFrame contain records of the temperature in Berlin from January to April:
+
+![Dataframe exploration](visualization-dataframe-temperature.png){width=600}
+
+> There are different options to access a column's records that can help you increase type safety when working with the Kandy and Kotlin DataFrame libraries together.
+> For more information, see [Access APIs](https://kotlin.github.io/dataframe/apilevels.html).
+>
+{type="tip"}
+
+## Create a line chart
+
+Let's create a line chart with the `df` DataFrame in Kotlin Notebook.
```

Comment:
Let's create a line chart in Kotlin Notebook using the `df` DataFrame from the previous section.